uber/config.py

```diff
@@ -131,6 +143,16 @@ def __getattr__(self, name):
                 return sa.localized_now() > date_setting
         elif name.startswith('HAS_') and name.endswith('_ACCESS'):
             return getattr(c, name.split('_')[1]) in sa.AdminAccount.access_set()
+        elif name.endswith('_AVAILABLE'):
+            item_check = name.split('_', 1)[0]
```

Comment:
Ah, I think I see what you're saying - I misunderstood your previous comment.  You're saying that you would prefer the use of ``self`` when accessing properies which are actually defined as part of the ``Config`` class declaration, but you'd prefer to use ``c`` when accessing attributes which were set on the instance from the config file?  So even though ``self`` and ``c`` are literally the same object, you find it more self-documenting to say ``getattr(c, `` on the first line and ``getattr(self, `` on the second line because the first one is accessing a property and the second one is accessing a property?